src/platform_impl/linux/mod.rs

```diff
@@ -607,7 +618,10 @@ unsafe extern "C" fn x_error_callback(
             minor_code: (*event).minor_code,
         };
 
-        error!("X11 error: {:#?}", error);
+        // Don't log error.
+        if should_error {
+            error!("X11 error: {:#?}", error);
+        }
```

Comment:
Is logging an error when there are no error hooks not interesting?

Reply:
It's. I'll fix it.